docker/environment_variables/psql.env

```diff
@@ -0,0 +1,8 @@
+DB_HOST=db
+
+# PSQL data
+DB_DATABASE=squest
+DB_PASSWORD=squest
+DB_USER=squest
+POSTGRES_PASSWORD=psqlSuperP@ssword
```

Comment:
Maybe here we should keep only one file named db.env and use generic variable name like you did.
The only missing ones are MYSQL_ROOT_PASSWORD and POSTGRES_PASSWORD which are the same and could be renamed "DB_ADMIN_PASSWORD" or "DB_ROOT_PASSWORD".
Then the mapping is done in the compose file.
 What do you think?